packages/integration/lib/src/widgets/user/user_created_comment_listview.dart

```diff
@@ -0,0 +1,787 @@
+  }) : super(key: key);
+
+  // The user id for which the user feed is to be shown
+  final String uuid;
+  // Builder for post item
+  // {@macro post_widget_builder}
+  final LMFeedPostWidgetBuilder? postBuilder;
+  // {@macro context_widget_builder}
+  // Builder for empty feed view
+  final LMFeedContextWidgetBuilder? emptyFeedViewBuilder;
+  // Builder for first page loader when no post are there
+  final LMFeedContextWidgetBuilder? firstPageLoaderBuilder;
+  // Builder for pagination loader when more post are there
+  final LMFeedContextWidgetBuilder? paginationLoaderBuilder;
+  // Builder for error view when error occurs
+  final LMFeedContextWidgetBuilder? feedErrorViewBuilder;
+  // Builder for widget when no more post are there
+  final LMFeedContextWidgetBuilder? noNewPageWidgetBuilder;
+
+  @override
+  State<LMFeedUserCreatedCommentListView> createState() =>
+      _LMFeedUserCreatedCommentListViewState();
+}
+
+class _LMFeedUserCreatedCommentListViewState
+    extends State<LMFeedUserCreatedCommentListView> {
+  ValueNotifier<bool> rebuildPostWidget = ValueNotifier(false);
+  int _page = 1;
+  static const int pageSize = 10;
+  final PagingController<int, LMCommentViewData> _pagingController =
+      PagingController(firstPageKey: 1);
+  bool userPostingRights = true;
+  LMFeedThemeData? feedThemeData;
+  final ValueNotifier postUploading = ValueNotifier(false);
+  final LMFeedUserCreatedCommentBloc _userCreatedCommentBloc =
+      LMFeedUserCreatedCommentBloc();
+  final Map<String, LMPostViewData> _posts = {};
+
+  @override
+  void initState() {
+    super.initState();
+    userPostingRights = checkPostCreationRights();
+    addPageRequestListener();
+    _userCreatedCommentBloc.add(
+      LMFeedUserCreatedCommentGetEvent(
+        uuid: widget.uuid,
+        page: _page,
+        pageSize: pageSize,
+      ),
+    );
+  }
+
+  bool checkPostCreationRights() {
+    final MemberStateResponse memberStateResponse =
+        LMFeedUserLocalPreference.instance.fetchMemberRights();
+    if (!memberStateResponse.success || memberStateResponse.state == 1) {
+      return true;
+    }
+    final memberRights = LMFeedUserLocalPreference.instance.fetchMemberRight(9);
+    return memberRights;
+  }
+
+  void addPageRequestListener() {
+    _pagingController.addPageRequestListener(
+      (pageKey) {
+        _userCreatedCommentBloc.add(
+          LMFeedUserCreatedCommentGetEvent(
+            uuid: widget.uuid,
+            page: _page,
+            pageSize: pageSize,
+          ),
+        );
+      },
+    );
+  }
+
+  void refresh() => _pagingController.refresh();
+
+  // This function updates the paging controller based on the state changes
+  void updatePagingControllers(UserCreatedCommentLoadedState state) {
+    if (state.comments.length < 10) {
+      _pagingController.appendLastPage(state.comments);
+      _posts.addAll(state.posts);
+    } else {
+      _pagingController.appendPage(state.comments, _page + 1);
```

Comment:
send the pagekey in state that you received in the event and use that pageKey here